### PR TITLE
davix 0.7.5

### PR DESCRIPTION
**Pull request by gbitzes**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/davix.rb [hunk 1]
```diff
@@ -22,8 +22,10 @@ class Davix < Formula
   def install
     ENV.libcxx
 
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    cp "release.cmake", "version.cmake"
```
Comment (fxcoudert) [inline Formula/davix.rb]: Why is the build process changing?

Comment (gbitzes) [reply Formula/davix.rb]: There's still a slight bug in the machinery which generates version numbers in 0.7.5: The existence of version.cmake should not have been necessary, but cmake complains if it is not.

0.7.4 could not be built due to this: https://github.com/Homebrew/homebrew-core/pull/41640

I'll make sure to fix this in 0.7.6 (I maintain davix), in the meantime I'm adding a workaround here.

Comment (fxcoudert) [reply Formula/davix.rb]: Great, thanks for explaining. One tiny other thing: we tend to prefer to build from within the build directory, to maintain our formulas homogeneous:

```
mkdir "build" do
  system "cmake", "..", *std_cmake_args
  system "make", "install"
```

Comment (gbitzes) [reply Formula/davix.rb]: Ah, sure. I've updating the formula to use in-source build.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @gbitzes

**Pull request closed without merging**